.github/workflows/cherryPick.yml

```diff
@@ -190,15 +203,15 @@ jobs:
         if: fromJSON(steps.cherryPick.outputs.HAS_CONFLICTS)
         run: |
           gh pr edit --add-assignee "${{ github.actor }},${{ steps.getCPMergeCommit.outputs.MERGE_ACTOR }}"
-          ORIGINAL_PR_AUTHOR="$(gh pr view ${{ github.event.inputs.PULL_REQUEST_NUMBER }} --json author --jq .author.login)"
+          ORIGINAL_PR_AUTHOR="$(gh pr view ${{ steps.GetPRInfo.outputs.PR_NUMBER }} --json author --jq .author.login)"
```

Comment:
We should just us the URL here, because that way it will work in other repos without having to add the `--repo` flag